package.json

```
@@ -39,15 +39,15 @@
     "husky": "^4.2.3",
     "jest": "^24.9.0",
     "prettier": "^1.18.2",
-    "react": "^16.9.0",
-    "react-native": "0.60.5",
+    "react": "^16.11.0",
+    "react-native": "0.62.2",
     "react-test-renderer": "^16.9.0",
     "ts-jest": "^24.0.2",
     "typescript": "~3.7.5"
   },
   "peerDependencies": {
-    "react": "^16.8.0",
-    "react-native": "^0.59.0"
+    "react": "^16.11.0",
+    "react-native": "^0.62.2"
```

It doesn't look like we are introducing anything that would break support for the react and react-native versions that we previously had here. I think we can afford to be quite lenient:

```suggestion
    "react": ">=16.8.0",
    "react-native": ">=0.59.0"
```

---

It is breaking the jest testing for application thats using react-native v0.60+ due to path changes or react-native modules. Attach issue contains the description of the problem.

---

Let me test and see if this still fixes my problem